tests/Unit/CartTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+use PHPUnit\Framework\TestCase;
+use App\Cart;
+
+class CartTest extends TestCase
+{
+    /**
+     * If item is added to the cart
+     *
+     * @covers Cart::getItems
+     */
+    public function testAddItem()
+    {
+        $cart = new Cart();
+        $cart->addItem('foo');
+        $this->assertEquals(array('foo'), $cart->getItems());
+        $this->assertNotEquals(array('bar'), $cart->getItems());
+    }
+
+    /**
+     * Remove an item from the cart
+     *
+     * @covers Cart::getItems
+     */
+    public function testRemoveItem()
+    {
+        $cart = new Cart();
+        $cart->removeItem(array('foo'));
```

Comment:
What is the effect of removing an item, when there wasn't an item in the cart before?